packages/SwingSet/test/basedir-terminate/bootstrap.js

```diff
@@ -0,0 +1,20 @@
+/* global harden */
+
+import { E } from '@agoric/eventual-send';
+
+export function buildRootObject() {
+  return harden({
+    async bootstrap(vats, devices) {
+      const vatMaker = E(vats.vatAdmin).createVatAdminService(devices.vatAdmin);
+      const vat = await E(vatMaker).createVatByName('dude');
+      const before = await E(vat.root).dude();
+      console.log(`success result ${before}`);
```

Comment:
let's comment out the test's `console.log`, once the test is working correctly, I think it just adds noise to the test run (I'm guilty of this all over the place, but for new tests I'm trying to atone)

Reply:
Yup, it's always nice to have the reassurance but once it's in CI it's just noise.